src/Tasks/Backup/FileSelection.php

```diff
@@ -159,7 +159,7 @@ protected function sanitize($paths): Collection
                 return $path === '';
             })
             ->flatMap(function ($path) {
-                return glob(str_replace('*', '{.[!.],}*', $path), GLOB_BRACE);
+                return strpos($path, '*') !== false && defined('GLOB_BRACE') ? glob(str_replace('*', '{.[!.],}*', $path), GLOB_BRACE) :  glob($path);
```

Comment:
Could you make this more readable by moving that conditional statement to a function that returns a boolean. Give that function a nice name.

Reply:
maybe more readable if a function return all the array